dotnet/samples/KernelSyntaxExamples/Example15_TextMemorySkill.cs

```diff
@@ -0,0 +1,280 @@
+    private static async Task<IMemoryStore> CreateSampleSqliteMemoryStoreAsync()
+    {
+        IMemoryStore store = await SqliteMemoryStore.ConnectAsync("memories.sqlite");
+        return store;
+    }
+
+    private static async Task<IMemoryStore> CreateSampleDuckDbMemoryStoreAsync()
+    {
+        IMemoryStore store = await DuckDBMemoryStore.ConnectAsync("memories.duckdb");
+        return store;
+    }
+
+    private static IMemoryStore CreateSampleAzureCognitiveSearchMemoryStore()
+    {
+        IMemoryStore store = new AzureCognitiveSearchMemoryStore(TestConfiguration.ACS.Endpoint, TestConfiguration.ACS.ApiKey);
+        return store;
+    }
+
+    private static IMemoryStore CreateSampleQdrantMemoryStore()
+    {
+        IMemoryStore store = new ChromaMemoryStore(TestConfiguration.Chroma.Endpoint, ConsoleLogger.Logger);
+        return store;
+    }
+
+    private static IMemoryStore CreateSampleChromaMemoryStore()
+    {
+        IMemoryStore store = new QdrantMemoryStore(TestConfiguration.Qdrant.Endpoint, 1536, ConsoleLogger.Logger);
+        return store;
+    }
+
+    private static IMemoryStore CreateSamplePineconeMemoryStore()
+    {
+        IMemoryStore store = new PineconeMemoryStore(TestConfiguration.Pinecone.Environment, TestConfiguration.Pinecone.ApiKey, ConsoleLogger.Logger);
+        return store;
+    }
+
+    private static IMemoryStore CreateSampleWeaviateMemoryStore()
+    {
+        IMemoryStore store = new WeaviateMemoryStore(TestConfiguration.Weaviate.Endpoint, TestConfiguration.Weaviate.ApiKey);
+        return store;
+    }
+
+    private static async Task<IMemoryStore> CreateSampleRedisMemoryStoreAsync()
+    {
+        string configuration = TestConfiguration.Redis.Configuration;
+        await using ConnectionMultiplexer connectionMultiplexer = await ConnectionMultiplexer.ConnectAsync(configuration);
+        IDatabase database = connectionMultiplexer.GetDatabase();
+        IMemoryStore store = new RedisMemoryStore(database, vectorSize: 1536);
+        return store;
+    }
+
+    private static async Task<IMemoryStore> CreateSamplePostgresMemoryStoreAsync()
+    {
+        NpgsqlDataSourceBuilder dataSourceBuilder = new(TestConfiguration.Postgres.ConnectionString);
+        dataSourceBuilder.UseVector();
+        await using NpgsqlDataSource dataSource = dataSourceBuilder.Build();
+        IMemoryStore store = new PostgresMemoryStore(dataSource, vectorSize: 1536, schema: "public");
+        return store;
+    }
+
+    private static IMemoryStore CreateSampleKustoMemoryStore()
+    {
+        var connectionString = new Kusto.Data.KustoConnectionStringBuilder(TestConfiguration.Kusto.ConnectionString).WithAadUserPromptAuthentication();
+        IMemoryStore store = new KustoMemoryStore(connectionString, "MyDatabase");
+        return store;
+    }
+
+    private static async Task RunWithStoreAsync(IMemoryStore memoryStore, CancellationToken cancellationToken)
+    {
+        var logger = ConsoleLogger.Logger;
```

Comment:
I just tried to run example with Chroma and this logger outputs too many messages with transient errors and warnings, so console window is overloaded and it's hard to find actual function results. What if we set `NullLogger.Instance` by default and provide a comment that for detailed logs it's possible to use `ConsoleLogger.Logger`? 

It applies for loggers on Kernel and each memory store level.

Reply:
Aside: I've been updating ConsoleLogger locally to expose the LogFactory so I can create loggers more easily and tune filters per category/name/type. I'll try and get that pulled out of a draft and merged with the recent ILogger changes.